tests/tests_api/test_language_and_level.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from rest_framework import status
+
+from api.models import LanguageAndLevel
+
+
+@pytest.mark.parametrize("language", [None, "en", "fr"])
+def test_languages_and_levels_list(admin_client, language):
+    queryset = LanguageAndLevel.objects.all()
+
+    if language:
+        queryset = queryset.filter(language=language)
+        query_string = f"?language={language}"
+    else:
+        query_string = ""
+
+    response = admin_client.get(f"/api/languages_and_levels/{query_string}")
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json() == [
+        {
+            "id": language_and_level.id,
+            "language": {
+                "id": language_and_level.language.id,
+                "name": language_and_level.language.name,
+            },
+            "level": str(language_and_level.level),
```

Comment:
hm... should we rely on `__str__()` method of `LanguageLevel`? I would prefer `language_and_level.level.id` here, like you do in day and time slots. Otherwise, if we change the `__str__()` on `LanguageLevel`, we'll get a sudden bug, although `id` won't change.

Reply:
Agree, we shouldn't rely on str() totally :)